lib/generators/matestack_component/templates/matestack_component.scss.erb

```diff
@@ -0,0 +1 @@
+// your styles for the component <%= class_name %> go here
```

Comment:
I'm not sure if everyone wants to use scss. plan css should be supported.

Reply:
in addition

Reply:
I'm not sure how to decide here. Should we add a single `css` file for each created component, should we give the option to choose from css, sass or scss, or just remove styling from the scaffolder for now @jonasjabari ?

Reply:
give the option to choose from css, sass or scss :)